code/modules/power/supermatter/supermatter.dm

```diff
@@ -207,6 +213,8 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	RegisterSignal(src, COMSIG_ATOM_BSA_BEAM, .proc/call_explode)
 
 	soundloop = new(list(src), TRUE)
+	if(ispath(psyOverlay))
+		psyOverlay = new psyOverlay()
```

Comment:
```suggestion
	if(ispath(psyOverlay))
		psyOverlay = new psyOverlay()
	else
		stack_trace("Supermatter created with non-path psyOverlay variable. This can break things, please fix.")
		psyOverlay = new()
```

Reply:
yep